### DISCUSSION TITLE
支持图片应用

### DISCUSSION THREAD

**Discussion by Jengro777** [ChatGPTNextWeb/NextChat#3246] [Ideas]
有时候我需要让AI 帮助我微调图片, 甚至直接让AI 把图片上的中文 转换成英文后, 将翻译优化好的 图片输出给我

这个想法可以实现吗?

**Comment by Jengro777** [comment]
参考: https://demo6.goamzai.com/draw

**Comment by eric-easyhealth** [comment]
NextWeb 好像不支持用 gpt模型直接画图吧？ 我找了半天没有找到这个功能
